db/db_bolt.go

```diff
@@ -396,6 +407,9 @@ func (b *BoltDB) Insert(name []byte, key uint64, value []byte) error {
 		}
 	}
 	if err != nil {
+		if errors.Is(err, syscall.ENOSPC) {
+			log.L().Fatal("Failed to insert db.", zap.Error(err))
```

Comment:
also add into `Remove() and Purge()` func

Reply:
ok. fixed.